contracts/TokenManager.sol

```diff
@@ -27,17 +27,22 @@ contract TokenManager is Ownable {
 
     // Gets the details for the given token, if it is accepted
     /// @param _name 32-byte array value representation of the token symbol e.g. "WETH", "USDT"
-    function get(bytes32 _name) external view returns(Token memory) {
-        return tokens[_name];
+    function get(bytes32 _name) external view returns(Token memory token) {
```

Comment:
i see that there is a path test-failure case on L36 in `tokenManager.js`. could you also add a happy case or two as well, despite the fact this being a fairly low-complex loop.

Reply:
[this](https://github.com/the-standard/ibco/blob/571153408f5a9d313cdd9cf45dca7f122d505d18/test/tokenManager.js#L26-L33), no?

Reply:
yup, that will make it. was only looking at the PR changes and didn't see previous code.